doc/source/whatsnew/v0.24.0.txt

```diff
@@ -26,6 +26,33 @@ Backwards incompatible API changes
 
 .. _whatsnew_0240.api.datetimelike:
 
```

Comment:
add a ref to this section (this is a new one, like: _whatsnew_0240.api.datetimelike.normalize )